web_programming/get_links_of_url.py

```diff
@@ -0,0 +1,46 @@
+import re
+import urllib.parse
+import urllib.request
```

Comment:
Why use `urllib.request` when there is this awesome library `requests`?
Please take a look at that.